### PR TITLE
Support the `--ca-bundle` option on SSO commands

### PR DESCRIPTION
**Pull request by ashovlin**
*Issue #, if available:* #7602

*Description of changes:*
Before, when constructing the SSO-OIDC client used inside the `aws sso login`, `aws sso logout`, and `aws configure sso` commands, we were not setting the `verify` parameter. These commands did not honor the global `--ca-bundle` and `--no-verify-ssl` options.

Now, we plumbed through `parsed_globals` to set `verify` appropriately.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .changes/next-release/bugfix-sso-68327.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "``sso``",
+  "description": "Support the ``--ca-bundle`` option on SSO commands."
```
Comment (aemous) [inline .changes/next-release/bugfix-sso-68327.json:4]: Can we also add that we support the ``--no-verify-ssl`` option now?

**Review by aemous — CHANGES_REQUESTED** [review]
Looks good overall, can we also get a couple corresponding tests that include the ``--no-verify-ssl`` arg on the sso commands?

**Review by TheerapatSrisombat — COMMENTED** [review]
`import numpy as np

# กำหนดพิกัดเริ่มต้น
x = 19.3476535
y = 100.533892

# สร้างเมทริกซ์เพื่อแทนค่าต่างๆ
matrix = np.array([[x, y], [0, 1], [2, 3]])

# ตัวอย่างการคำนวณง่ายๆ
result = np.sum(matrix)

print(result)`

**Review by aemous — APPROVED** [review]

**Comment by aemous** [comment]
Looks good to me

File: .changes/next-release/bugfix-sso-68327.json (hunk 1 above)
Comment (ashovlin) [reply .changes/next-release/bugfix-sso-68327.json]: Added via https://github.com/aws/aws-cli/pull/9112/commits/7ea5e364410bc8e4c1ed63d7e87bf7f566b66816 (changelog message too).

**Review by hssyoo — APPROVED** [review]

**Pull request merged**